cmd/jaeger/internal/all-in-one.yaml

```diff
@@ -30,7 +30,8 @@ extensions:
   remote_sampling:
     # We can either use file or adaptive sampling strategy in remote_sampling
     file:
-      path: ./cmd/jaeger/sampling-strategies.json
+      path:
+      default_sampling_probability: 1
```

Comment:
Would it make sense as a field at the top level? Right now we have `file.default_sampling_probability` and `adaptive.initial_sampling_probability`, but their meaning is very similar - which probability to return in the absence of any better information.